src/pages/resultsView/pathwayMapper/ResultsViewPathwayMapper.tsx

```diff
@@ -318,12 +365,14 @@ export default class ResultsViewPathwayMapper extends React.Component<
     private dismissActiveToasts() {
         // Toasts are removed with delay
         setTimeout(() => {
-            this.activeToasts.forEach(tId => {
-                toast.dismiss(tId);
-            });
+            this.message = null;
```

Comment:
if we're using alerts instead of toasts, do we still need to dismiss them?    also, if we're getting rid of toasts (fine by me) lets clean out the toast code.

Reply:
Actually, dismissing here corresponds to setting the message as the default message which is "Welcome to PathwayMapper tab".

Reply:
It would be great if we could completely get rid of the `dismissMessage` function